ansible/group_vars/all

```diff
@@ -55,7 +55,8 @@ whisk:
 # configuration parameters related to support runtimes (see org.apache.openwhisk.core.entity.ExecManifest for schema of the manifest).
 # briefly the parameters are:
 #
-#   runtimes_registry: optional registry (with trailing slack) where to pull docker images from for runtimes and backbox images
+#   runtimes_registry: optional registry (with trailing slash) where to pull docker images from for default runtimes (in manifest)
+#   user_images_registry: optional registry (with trailing slash) where to pull docker images from for blackbox images
```

Comment:
Just to confirm wrt compatibility - If user was earlier configuring `runtimes_registry` which was used for "both" images from for runtimes and backbox images would now have to configure both `runtimes_registry` and `user_images_registry`. 

Or system would default to `runtimes_registry` if `user_images_registry` is not configured

Reply:
They are separately evaluated.
Earlier there was only one option to add `runtimes_registry` to default images. Blackbox images were only evaluated as is (no system registry applied).
This PR adds `user_images_registry` to apply blackbox images. If this is not set, blackbox images are evaluated by its own name.